app/presenters/order_presenter.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class OrderPresenter
+  def initialize(order)
+    @order = order
+  end
+
+  def total
+    @order.products.pluck(:price).sum
```

Comment:
Tutaj muszę pamiętać, że kiedy będzie można kupić więcej sztuk tych samych produktów, to będzie się musiało zmienić.

Reply:
dobra praktyka jest stworzenie sobie na to gdzies taska :)